### ISSUE TITLE
the results i produced is 94%, lower than the results that you mentioned in paper

### ISSUE THREAD

**Issue by WJ1214** [skumra/robotic-grasping#37]
Hi, this is a good work, i trained model on Cornell dataset by default arguments, but the best result in Cornell dataset is 94%, which is lower than the results that you mentioned in paper. How can i get the result in you paper?

**Comment by LIAGM** [comment]
This released code can not reproduce and achieve the performance which showed in the paper.
Even you use the pretrained model to evaluate both Cornell and Jacquard Dataset.
You still can not achieve the performance showed in paper.

**Comment by WJ1214** [comment]
> This released code can not reproduce and achieve the performance which showed in the paper.
> Even you use the pretrained model to evaluate both Cornell and Jacquard Dataset.
> You still can not achieve the performance showed in paper.

yes..., I have found it, if I use trained model which accuracy is 94% to evaluate Cornell, evaluation result is 91%....
it's wired....

**Comment by LIAGM** [comment]
And you can see there are some important change in the release 0.3.0 which may affect the result performance.
But the author did not retrain the Cornell dataset model, that means the original pretrain model is wrong for the new code.

**Comment by WJ1214** [comment]
> And you can see there are some important change in the release 0.3.0 which may affect the result performance.
> But the author did not retrain the Cornell dataset model, that means the original pretrain model is wrong for the new code.

I see.... the author deleted quality attribute of grasp in 0.3.0.....

**Comment by LIAGM** [comment]
I think the quality attribute is not so important.
The parameters changed in the  peak_local_max are more important and could cause different results.
This code is largely borrowed from GGCNN.
You can find some different settings like input size or other things.
Like the code in the inference/post_process.py do not change the normalization scalar according to the input size.
This will affect the performance to better results but in fact the accuracy is not so good.
Overall, I think there are many bugs in this project and the results in the paper is not convincing.

**Comment by silver60229** [comment]
> I think the quality attribute is not so important.
> The parameters changed in the peak_local_max are more important and could cause different results.
> This code is largely borrowed from GGCNN.
> You can find some different settings like input size or other things.
> Like the code in the inference/post_process.py do not change the normalization scalar according to the input size.
> This will affect the performance to better results but in fact the accuracy is not so good.
> Overall, I think there are many bugs in this project and the results in the paper is not convincing.

Hello, please are you a student or professor at National Taiwan University? I am currently a graduate student and I am also learning robotic-grasping, but I am a beginner and my laboratory does not have many resources to help me. 
If it is convenient to ask you some professional knowledge?Leave an email to ask you some questions? I am also from Taiwan. Thank you

**Comment by LIAGM** [comment]
> Hello, please are you a student or professor at National Taiwan University? I am currently a graduate student and I am also learning robotic-grasping, but I am a beginner and my laboratory does not have many resources to help me.
> If it is convenient to ask you some professional knowledge?Leave an email to ask you some questions? I am also from Taiwan. Thank you

Hi, I am research assistant at NTU and I am also new to the field of robotic grasping!
It's OK to email me to discuss your problems.
You can find my email in my github page.

**Comment by WJ1214** [comment]
> I think the quality attribute is not so important.
> The parameters changed in the peak_local_max are more important and could cause different results.
> This code is largely borrowed from GGCNN.
> You can find some different settings like input size or other things.
> Like the code in the inference/post_process.py do not change the normalization scalar according to the input size.
> This will affect the performance to better results but in fact the accuracy is not so good.
> Overall, I think there are many bugs in this project and the results in the paper is not convincing.

Alright, I just read the code of GGCNN, this project is same as GGCNN in most places...
